src/js/visual/configurator.js

```diff
@@ -912,6 +914,7 @@ ripe.Configurator.prototype._loadFrame = async function(view, position, options
         if (isReady) {
             await this._drawFrame(image, animate, duration);
         }
+        this.trigger("post_frame", view, position, options);

```

Comment:
Maybe we should include a boolean in the arguments saying if this was a no-op (the same applies to the mask when it doesn't actually load a new image).